lib/findExports.js

```diff
@@ -0,0 +1,203 @@
+import fs from 'fs';
+import path from 'path';
+
+import parse from './parse';
+import requireResolve from './requireResolve';
+
+function findESNamedExports(node) {
+  if (node.type !== 'ExportNamedDeclaration') {
```

Comment:
Some of these AST bits could still probably benefit from comments with examples of what they mean.